doc/source/ref/wavelets.rst

```diff
@@ -249,3 +249,143 @@ from plain Python lists of filter coefficients and a *filter bank-like* objects.
     ...         return [dec_lo, dec_hi, rec_lo, rec_hi]
     >>> filter_bank = HaarFilterBank()
     >>> myOtherWavelet = pywt.Wavelet(name="myHaarWavelet", filter_bank=filter_bank)
+
+
+``ContinuousWavelet`` object
+------------------
+
+.. class:: ContinuousWavelet(name)
+
+  Describes properties of a continous wavelet identified by the specified wavelet *name*.
+  In order to use a built-in wavelet the *name* parameter must be a valid
+  wavelet name from the :func:`pywt.wavelist` list.
```

Comment:
Somewhat off topic for this PR, but perhaps `wavelist` should have an argument called `type` that could be set to one of `{'discrete', 'continuous', 'both'} that would list only particular types of wavelets?  Otherwise the user will see many wavelets listed with no easy way to identify which are continuous vs. discrete.